ftests/019-cgconfig-uidgid_dperm_fperm.py

```diff
@@ -113,11 +125,12 @@ def teardown(config):
         else:
             Run.run(['sudo', 'userdel', USER])
             Run.run(['sudo', 'groupdel', GROUP])
-    except:
+    except (config.args.ContainerError, ValueError, RunError):
```

Comment:
Would it be cleaner to do this?

```
from container import ContainerError
...
except (ContainerError, ValueError, RunError):
```

Reply:
I did think about it, voted against it after `config.args.Container....` usage in the test cases. Will fix it.